assets/community/bounces.toml

```diff
@@ -50,3 +63,21 @@ AuthenticationFailed = [
 ]
 PolicyRelated = [
 ]
+ReputationBlock = [
+  "black.*list",
+  "block.*list",
+  "blocked",
+  "dnsbl",
+  "ip is rejected",
+  "policy rejection",
+  "rejected due to.*reputation",
+  "sender denied",
+  "spamrl",
+  "uceprotect",
+]
+SpamBlock = [
+  "High probability of spam",
+  "High scoring spam message has been dropped",
+  "This email from IP SENDERIP has been rejected\\. The email message was detected as spam"
+  "This email has been rejected\\. The email message was detected as spam"
```

Comment:
This file is invalid toml  Please be sure to actually test these rules before submission!

That said, it seems to me like this could be combined with the line above and simply match `The email message was detected as spam` as that string is present in both and is rather clear in intent.